src/modules/notifications/notifications.service.ts

```diff
@@ -35,6 +37,29 @@ export class NotificationsService {
     private userRepository: Repository<User>
   ) {}
 
+  async createGlobalNotifications(dto: CreateNotificationForAllUsersDto) {
+    const users = await this.userRepository.find();
+    try {
```

Comment:
there is no need for the try/cacth block. The interceptor will handle all non-specific errors appropriately

Reply:
Updated